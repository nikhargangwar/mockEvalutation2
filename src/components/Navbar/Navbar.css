.navbar{
    margin: 0px;
    width: 100%;;
    height: 8vh;
    background-color: #38497E;
    display: flex;
    align-items: center;
}

.navbar-text{
    display: flex;
    color: white;
    font-weight:lighter;
    font-size: large;
    width: 200px;
    justify-content: space-between;
    margin-left: 20px;
}
.navbar-text h2{
    display: flex;
    color: white;
    font-weight:300;
    
}

.record-text{
    font-weight:900 !important;
}
