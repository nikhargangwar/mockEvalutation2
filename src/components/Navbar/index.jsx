import './Navbar.css';
import React from 'react';

function Navbar() {
  return (
    <div className="navbar">
      <div className="navbar-text">
        <h2>My</h2>
        <br />
        <h2 className="record-text">Record</h2>
        <h2>Shelf</h2>
      </div>
    </div>
  );
}

export default Navbar;
