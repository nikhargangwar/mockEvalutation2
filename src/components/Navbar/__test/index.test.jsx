import React from 'react';
import {
  render,
} from '@testing-library/react';
import Navbar from '..';

describe('Navbar', () => {
  it('should render card when mounted', () => {
    const { container } = render(<Navbar />);
    expect(container).toMatchSnapshot();
  });
});
