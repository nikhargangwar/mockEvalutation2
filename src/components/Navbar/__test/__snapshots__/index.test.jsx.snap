// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render card when mounted 1`] = `
<div>
  <div
    class="navbar"
  >
    <div
      class="navbar-text"
    >
      <h2>
        My
      </h2>
      <br />
      <h2
        class="record-text"
      >
        Record
      </h2>
      <h2>
        Shelf
      </h2>
    </div>
  </div>
</div>
`;
